src/data.ml

```diff
@@ -155,6 +155,12 @@ module People = struct
     uri       = None;
     email     = None;
   }
+
+  let linse = {
+    Atom.name = "Stefanie Schirmer"
+    uri       = "https://linse.me"
+    email     = None;
+  }
```

Comment:
travis complains about this code -- line 160 needs a trailing `;`, 161 needs to have an `option` value and a `;` (`Some "https:/linse.me";`